### ISSUE TITLE
Rendering of oneway=yes

### ISSUE THREAD

**Issue by John07** [migurski/paperwalking#33]
From talk-de:

Rendering of oneway=yes

**Comment by goldfndr** [comment]
oneway=yes is rendered on some of the providers. Isn't this a provider issue?

**Comment by John07** [comment]
Yes, it is. But Walking Paper can choose the style they provide (in fact everyone can push a style change) and can also try to influence the provider so they're going to render oneway=yes. So you're partly right, but for the user of the website it doesn't matter who is responsible for this, they just want to have oneway=yes renderd.
